docs/lib/analytics/menu.json

```diff
@@ -7,6 +7,7 @@
     "identifyuser",
     "streaming",
     "storing",
-    "personalize"
+    "personalize",
+    "configure"
```

Comment:
"existingresource" ?

Reply:
This isn't reader-facing. I guess the goal here is a name that will be clear to a writer at a glance. Let's use "existingresources."

Reply:
Went with existing-resources.